contracts/zero-ex/contracts/test/samplers/SynthetixSampler.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Do we really need all of these Samplers duplicated from 0x-API? Will they all be tested here and not in 0xAPI?

Reply:
my intention was to start writing some sampler tests, but if we dont want to introduce them here we can remove them. for the forked integration tests, i was actually calling out to the samplers to get a sample to trade